javatests/arcs/core/analysis/ExpressionDependencyAnalyzerTest.kt

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.analysis
+
+import arcs.core.data.expression.PaxelParser
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class ExpressionDependencyAnalyzerTest {
+
+  @Test
+  fun literal_number() {
+    val expr = PaxelParser.parse("5")
+
+    val actual = expr.analyze()
```

Comment:
Piotr suggested something similar. We decided to keep the boilerplate to make it easier to step through the debugger in tests. I.e., we want to keep the tests DAMP instead of DRY.